Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -167,7 +171,13 @@ async def fetch_event(self, initial_offset: int = 0, event_type: str = '') -> As
                             if stripped_line:
                                 events_fetched += 1
                                 try:
-                                    yield json.loads(stripped_line)
+                                    streaming_event = json.loads(stripped_line)
+                                    event_metadata = streaming_event.get('metadata', {})
+                                    event_creation_time = event_metadata.get('eventCreationTime', 0) / 1000
```

Comment:
I think it is worth adding a log if eventCreationTime is not present and 0 was used otherwise it will be very hard to understand why events are not coming in. In the log print the whole json.

Reply:
done